models/wrf/shell_scripts/driver.csh

```diff
@@ -372,6 +372,13 @@ while ( 1 == 1 )
 
    echo "ready to integrate ensemble members"
 
+   # Removing old start_member and done_member diagnostics
+   if ( -e ${RUN_DIR}/start_member_1) then
+      ${REMOVE} ${RUN_DIR}/start_member_*  \
+                ${RUN_DIR}/done_member_*
+   endif
+
+
```

Comment:
If first time through the driver.csh, there could be lingering start_member or done_member files from the prior init_ensemble_var.csh run. Need to get rid of these or the driver.csh script starts transferring files that do not exist because WRF integration is not yet completed.

Reply:
@mgharamti  I know you are super busy, but as a regular WRF user, would be worth for you to look this over, especially commit 63a5748de6f7fd775ee58bcf76d23131fbe6a97c .   This commit changes some of the wrf shell scripting which extends outside the tutorial.   I think it is OK and just a more simple way of linking the DART input.nml information to the WRF case.  No rush on this.  I think this is about as much as I want to change for foreseeable future.  Thanks @mjs2369 for looking at this too.   If everything looks OK please include in next DART release.

Reply:
@braczka I've looked over these changes and I agree that the WRF shell scripting needs to link ${RUN_DIR}/input.nml as opposed to ${TEMPLATE_DIR}/input.nml.template . I think this is ready to merge. 

But there's no rush on this release (the bug-fix that Helen implemented is developer code only), so I also think that we should wait for @mgharamti  to take a look if he has the time. If not, we can just go ahead and release sometime next week.